components/treeview/index.css

```diff
@@ -301,9 +301,20 @@
 }
 
 /* ***** VARIANTS ***** */
+.spectrum-TreeView--quiet {
+	--spectrum-treeview-item-background-color-selected: var(--highcontrast-treeview-item-background-color-selected, var(--mod-treeview-item-background-color-quiet-selected, var(--spectrum-treeview-item-background-color-quiet-selected)));
+	--mod-treeview-item-border-color-selected: var(--highcontrast-treeview-item-border-color-selected, var(--mod-treeview-item-border-color-quiet-selected, var(--spectrum-treeview-item-border-color-quiet-selected)));
```

Comment:
This has been changed to be prefixed with `--mod` as in `main`: https://github.com/adobe/spectrum-css/blob/main/components/treeview/index.css#L114

If `--spectrum-treeview-item-border-color-selected` is used instead of `--mod-treeview-item-border-color-selected`, the transparent border color does not come through